src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -108,14 +94,16 @@ public AndroidTargetConfiguration( ProcessPaths paths, InternalProjectConfigurat
 
         Path clangguess = Paths.get(this.ndk, "toolchains", "llvm", "prebuilt", hostPlatformFolder, "bin", "clang");
         this.clang = Files.exists(clangguess) ? clangguess : null;
+
+        projectConfiguration.setBackend(Constants.BACKEND_AARCH64);
```

Comment:
This won't work on 20.1, as it requires https://github.com/oracle/graal/commit/879a752eb9bae1aefa1576370414c668ebf0ec79

Reply:
https://github.com/gluonhq/substrate/pull/615 should fix this.